app/helpers/application_helper.rb

```diff
@@ -456,7 +456,7 @@ def preloaded_json
   end
 
   def client_side_setup_data
-    service_worker_url = Rails.env.development? ? 'service-worker.js' : Rails.application.assets_manifest.assets['service-worker.js']
+    service_worker_url = 'service-worker.js'
```

Comment:
Any reasons this is not a constant?

Reply:
Alternatively we could just inlined in `setup_data` - does that work?